LeagueToolkit/LeagueToolkit.csproj

```diff
@@ -1,28 +1,31 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <!-- project settings -->
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <TargetFrameworks>netstandard2.1;net5.0;net6.0</TargetFrameworks>   
```

Comment:
Well not directly. But the implementations can also change between versions. For example, I would like to add support for netstandard2.0 later, so that the library can be used under the .net framework (which is currently not possible).

And at the end of the day it doesn't hurt and depending on the framework of the application the appropriate version is used. I don't see any real disadvantage (except maybe build time which is not really relevant for this project).

But otherwise, of course, I can remove it. But for now I wait for your answer.

Reply:
The argument for netstandard2.0 I can understand, which could also add potential value for those consumers who cannot comply with netstandard2.1.
However, as netstandard2.1 is like a base set for everything .net3+, I really don't see a point to add net5.0 and net6.0 targets at the moment (otherwise, why not do .net core3.1 too?). At the end of the day, that's one of the design aspects of the net standard: To not have to build for every target framework individually, but being able to rely on one single API set.

Reply:
its removed :)